### PR TITLE
First part of fixes from mergeduty

### PR DESCRIPTION
**Pull request by MihaiTabara**
* adds instructions for manually reconfig stuff
* add missing information on how to connect to Bouncer
* fix urls and commits to have better examples for future diffs 
* add references to the blocking bug to Net Services when it comes to ftl hooks
* fixes all hrefs in the mergeduty docs

### CODE REVIEW

**Comment by MihaiTabara** [comment]
For the record, I'll address the following in a follow-up PR:
* hg/.hgrc default push in the esr52 and the bump-and-tag central section
* a note on possibly moving logs
* fixing Fennec templates for pushing the apk
* misc others

File: docs/mergeduty/howto.md [hunk 1]
```diff
@@ -124,12 +133,12 @@ when: this can happen before relman request migration. e.g. the same day but in
 1. The reconfiguration is triggered by a cron job every hour exactly on the hour (`0 * * * *` in crontab). Your options are:
    * Wait for the reconfig to happen via cron.
    * Ask buildduty to manually trigger it.
-   * Run it yourself, see [below](#appendix-perform-a-manual-reconfig) for instructions
+   * Run it yourself, see [how-to-manually-perform-reconfig](https://github.com/mozilla-releng/releasewarrior-2.0/blob/master/docs/misc-operations/manually-perform-bb-reconfig-generate-partials.md) for instructions
```
Comment (mozbhearsum) [inline docs/mergeduty/howto.md]: Do we still need this, and the other buildbot-config sections of this doc?

Comment (MihaiTabara) [reply docs/mergeduty/howto.md]: Sounds like bbconfigs gecko_version is useless now, I agree. But we still need it until esr52 dies, dont't we? Actually, to be fully honest, I never understood gecko_versions.json in bbconfigs' role.

Comment (mozbhearsum) [reply docs/mergeduty/howto.md]: I think it controls some builder additiona/removal. In any case, now that we won't be doing any more merges involving Buildbot - that's not relevant for merge day.

**Comment by MihaiTabara** [comment]
@lundjordan I still have to address the actions within mozharness, I didn't forget. That's coming up in a separate bug/patch possible PR to warrior. Also, I've just addressed your https://bugzilla.mozilla.org/show_bug.cgi?id=1441631#c5

**Pull request merged**